core/web/presenters/job.go

```diff
@@ -525,6 +525,8 @@ func NewJobResource(j job.Job) *JobResource {
 		resource.BootstrapSpec = NewBootstrapSpec(j.BootstrapSpec)
 	case job.Gateway:
 		resource.GatewaySpec = NewGatewaySpec(j.GatewaySpec)
+	case job.LegacyGasStationServer, job.LegacyGasStationSidecar:
+		// unsupported
```

Comment:
Is this OK? Or an oversight to correct?

Reply:
cc @mateusz-sekara @makramkd @RensR

Reply:
Removing this should be OK, functionality is equivalent right?

Reply:
Yeah this change aims to be equivalent while satisfying the linter.

But was it an oversight that support was not added in the first place? Should we create a ticket?

Reply:
I think it was intentional that it's unsupported. cc @jinhoonbang

Reply:
@jinhoonbang is an owner of this GasStation I believe ;)